runtime/inspector.rs

```diff
@@ -414,6 +414,13 @@ pub struct DenoInspector {
   pub debugger_url: Option<String>,
 }
 
+impl DenoInspector {
+  pub fn is_active(&self) -> bool {
```

Comment:
Nit: it might be good to be less ambiguous and call this method `has_active_sessions()` or `has_connected_clients()` even.

Reply:
Renamed to `has_active_sessions()`